e2e/dractions/actions.go

```diff
@@ -45,7 +43,8 @@ func EnableProtection(ctx types.Context) error {
 		return err
 	}
 
-	log.Debugf("Workload running on cluster %q", clusterName)
+	log.Infof("Protecting workload running in cluster %q (app namespace: %q, management namespace: %q)",
```

Comment:
I think we can remove "running" from all the logs except the validation suite. "Protecting workload in cluster xxx" is enough.

Reply:
ack, shall I take this up in my next pr ie adding INFO logs before and after deploy, enable, failover, relocate, disbale, undeploy?

Reply:
Since this issue already exists (at least for this log) it is fine.